terraform/aws_vpc/variables.tf

```diff
@@ -0,0 +1,22 @@
+variable "aws_profile" {
+  type        = "string"
+  description = "Name of the AWS profile used for authentication."
+  default     = "chef-engineering"
+}
+
+variable "aws_region" {
+  type        = "string"
+  description = "Name of the AWS region to create instances in."
+  default     = "us-west-1"
```

Comment:
If we're going to pick a default, we should pick `us-west-2` unless there is a technical reason not to.

Reply:
I switched the default from `us-west-2` to `us-west-1` for this first iteration because the initial focus was enabling developers to easily run the tests themselves and `us-west-2` is perpetually hitting the VPC limit (i.e. 5) which prevents the developer from creating their own.